### DISCUSSION TITLE
New HOOBS install no accessories

### DISCUSSION THREAD

**Discussion by daviddeacon** [dgreif/ring#793] [Q&A]
Hi,

I've just installed the ring bridge 9.22.1 in HOOBS and linked my ring account, but I'm getting no accessories showing up.

Is there something I'm missing as part of the initial setup? or should accessories get automatically added?

Thanks,
David.

**Comment by dgreif** [comment]
Do you happen to have a Ring Alarm that is not powered up or on cellular backup?

**Reply by daviddeacon** [reply]
I do have a ring alarm, it is powered on, does appear to have cellular backup.

**Reply by dgreif** [reply]
If it is on cellular backup, the plugin has to wait until it is connected to a real internet connection before proceeding. It's not possible to get the list of devices from the alarm while it is only in cellular back

**Reply by daviddeacon** [reply]
I don't think it's in cellular backup mode it's connected to wifi and has an IP address that's part of the network. It's a Gen 2 ring alarm if that makes any difference.

**Reply by dgreif** [reply]
Can you post the logs from the ring plugin during the HOOBS startup process?  There might be some hints in there
